Blockzilla/BrowserViewController.swift

```diff
@@ -185,6 +185,11 @@ class BrowserViewController: UIViewController {
         containWebView()
         createHomeView()
         createURLBar()
+        
+        // Listen for request desktop site notifications
+        let nc = NotificationCenter.default.addObserver(forName:Notification.Name(rawValue: UIConstants.strings.requestDesktopNotification), object:nil, queue:nil)  { _ in
```

Comment:
Do we need to cleanup this observer? I haven't used the new API

Reply:
My understanding is that since iOS 9, observers do not need to be cleaned up, as they are automatically removed when the controller gets de-allocated. [Here](https://developer.apple.com/library/content/releasenotes/Foundation/RN-Foundation/#10_11NotificationCenter) is some more info on the topic.